cetlvast/suites/docs/examples/example_07_polymorphic_alloc_deleter.cpp

```diff
@@ -148,3 +231,34 @@ TEST(example_07_polymorphic_alloc_deleter, example_usage_1)
 
     //![example_usage_1]
 }
+
+TEST_F(example_07_polymorphic_alloc_deleter, example_usage_2)
+{
+    //![example_usage_2]
+
+    cetl::pmr::polymorphic_allocator<MyObject> alloc{cetl::pmr::new_delete_resource()};
+
+    auto obj0 = cetl::pmr::InterfaceFactory::make_unique<MyObject>(alloc, "obj1", 4U);
+
+    auto obj1 = cetl::pmr::InterfaceFactory::make_unique<IIdentifiable>(alloc, "obj1", 4U);
```

Comment:
This is new - we make `MyObject` instance as before, BUT we return `IIdentifiable` interface. Note that 'IIdentifiable` has `protected` destructor.